nixos/modules/config/zram.nix

```diff
@@ -92,6 +103,29 @@ in
           as there's no gain from keeping them in RAM.
         '';
       };
+
+      recommendedTuningSettings = lib.mkOption {
+        default = false;
```

Comment:
Maybe we want to enable this by default then?

Reply:
alright!